sdcm/logcollector.py

```diff
@@ -1111,6 +1136,18 @@ def get_aws_instances_by_testid(self):
                 instance=instance,
                 global_ip=instance['PublicIpAddress'],
                 tags={**self.tags, "NodeType": "loader", }))
+        if self.params["use_cloud_manager"]:
```

Comment:
You can remove this `if` because `filtered_instances['siren_manager_nodes']` is empty if `use_cloud_manager` is false